Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=c247ed6f5205f9feebd276c4cbe45018b10f19fa
Commit:     c247ed6f5205f9feebd276c4cbe45018b10f19fa
Parent:     d5337debacc00591b3f81fc3c982b40af7de1ab6
Author:     Clemens Fruhwirth <clemens@endorphin.org>
AuthorDate: Wed Jan 7 11:43:48 2009 +0100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Wed Jan 7 11:43:48 2009 +0100

    ALSA: hda - Fix typos for AD1882 codecs
    
    Fixed typos of codec-id checks for AD1882/AD1882A.
    
    Cc: stable@kernel.org
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

 sound/pci/hda/patch_analog.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/pci/hda/patch_analog.c b/sound/pci/hda/patch_analog.c
index 13e4d91..2e7371e 100644
--- a/sound/pci/hda/patch_analog.c
+++ b/sound/pci/hda/patch_analog.c
@@ -4263,13 +4263,13 @@ static int patch_ad1882(struct hda_codec *codec)
 	spec->num_adc_nids = ARRAY_SIZE(ad1882_adc_nids);
 	spec->adc_nids = ad1882_adc_nids;
 	spec->capsrc_nids = ad1882_capsrc_nids;
-	if (codec->vendor_id == 0x11d1882)
+	if (codec->vendor_id == 0x11d41882)
 		spec->input_mux = &ad1882_capture_source;
 	else
 		spec->input_mux = &ad1882a_capture_source;
 	spec->num_mixers = 2;
 	spec->mixers[0] = ad1882_base_mixers;
-	if (codec->vendor_id == 0x11d1882)
+	if (codec->vendor_id == 0x11d41882)
 		spec->mixers[1] = ad1882_loopback_mixers;
 	else
 		spec->mixers[1] = ad1882a_loopback_mixers;
